modules/nw-secondary-ext-gw-status.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/configuring-secondary-external-gateway.adoc
+
+:_content-type: PROCEDURE
+[id="nw-secondary-ext-gw-status_{context}"]
+= View the status of a secondary external gateway
+
+You can view the status of a secondary external gateway that is configured for your cluster. The `status` field for the `AdminPolicyBasedExternalRoute` custom resource reports recent status messages whenever you update the resource, subject to a few limitations:
+
+- Namespaces impacted are not reported in status messages
+- Pods selected as part of a dynamic next hop configuration do not trigger status updates as a result of pod lifecycle events, such as pod termination
+
+.Prerequisites
+
+* You installed the OpenShift CLI (`oc`).
+* You are logged in to the cluster with a user with `cluster-admin` privileges.
+
+.Procedure
+
+* To access the status logs for a secondary external gateway, enter the following command:
++
+[source,terminal]
+----
+$ oc describe adminpolicybasedexternalroutes.k8s.ovn.org <name>
```

Comment:
@jab-rh Something is wrong here, describe will describe the policy rather than show it's status, replaced describe with status no go, not sure what the correct syntax is here, but it doesn't report the output suggested on L38.

Reply:
@tshefi, would it be `get`?

```
oc get adminpolicybasedexternalroutes <name> -o yaml
```

And then in the `status` part of the returned object?